libs/core/langchain_core/callbacks/manager.py

```diff
@@ -2169,3 +2246,185 @@ def _configure(
                 ):
                     callback_manager.add_handler(var_handler, inheritable)
     return callback_manager
+
+
+async def adispatch_custom_event(
```

Comment:
should we add ..versionadded for all the new methods

Reply:
Yes we should!